downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1432 @@
+
+   public isolated function 'start() returns error? {
+   }
+
+   public isolated function gracefulStop() returns error? {
+   }
+
+   public isolated function immediateStop() returns error? {
+   }
+
+   public isolated function detach(service object {} s) returns error? {
+   }
+
+   public isolated function attach(service object {} s, string[]|string? name = ()) returns error? {
+   }
+
+   public function init() returns error? {
+      ...
+      // object init failed due to some reason
+      if (object_init_failed) {
+         return error("Listener initialization failed");
+      }
+   }
+}
+```
+
+**Previous Syntax:**
+```ballerina
+listener lsn = new Listener();
+
+public class Listener {
+   ...
+   ...
+   // listener life cycle methods
+
+   public function init() {
+      ...
+      // object init failed due to some reason
+      if (object_init_failed) {
+         panic error("Listener initialization failed");
+      }
+   }
+}
+```
+
+##### Referring Lang Library Modules Without Using Quoted Identifiers
+
+Lang library module prefixes can now be used without the initial quote. For example, both the approaches below are now supported.
+
+```ballerina
+int a = int:sum(1, 2);
+```
+
+```ballerina
+int a = ‘int:sum(1, 2);
+```
+
+The way of overriding a langlib module is also relaxed now. For example, both the approaches below are now supported.
+
+```ballerina
+import somePackage.someModule as string;
+```
+
+and
+
+```ballerina
+import somePackage.someModule as `string;
+```
+
+##### Improved Lang Library Methods
+
+###### Introduction of the `includes` Method
+
+A new lang library method named `includes`, which tests whether a `string` value includes another `string` value has been added to the `ballerina/lang.string` library. It accepts an optional second argument to indicate the index to start searching from. It returns `true` only if the string contains the other string at an index greater or equal to the start index. The value `false` will be returned otherwise.
+
+```ballerina
+string str = "Ballerina Programming Language";
+boolean contains = str.includes("Language", 10);
+```
+
+###### Introduction of the `sleep` Method
+
+A new `sleep` method, which pauses the execution of the current strand for a specified time in seconds has been added to the `ballerina/lang.runtime` library.
+
+```ballerina
+runtime:sleep(2.0);
+```
+
+###### Introduction of the `getStackTrace` Method
+
+A new `getStackTrace` method has been introduced in the `ballerina/lang.runtime` library to get a stack trace for the current call stack for the specified execution point. It returns an array of stack frames. 
+
+```ballerina
+runtime:StackFrame[] stackFrames = runtime:getStackTrace();
+```
+
+###### Introduction of the `Cloneable` Type
+
+A new `Cloneable` type has been introduced to the `ballerina/lang.value` library. This is a cyclic union type. This type represents the values on which `clone` and `cloneReadOnly` can be applied on.
```

Comment:
Shouldn't this be 
```suggestion
A new type `Cloneable` has been introduced to the `ballerina/lang.value` library. This is a cyclic union type. This type represents the values on which `clone` and `cloneReadOnly` can be applied on.
```

Reply:
IMO, saying "A new `Cloneable` type" reads like `Cloneable` is a kind of type and we've introduced a new type that is of this kind.

Reply:
The idea is the same both ways.

Reply:
Changed to "a new type named `Clonable`".